File: client/src/app/_models/notification.ts
export interface Notification{
    id: number;
    title: string;
    isRead: boolean;
    createdAt: Date;
    readAt?: Date;
}

export interface NotificationDetailed{
    id: number;
    title: string;
    content: string;
    createdAt: Date;
    readAt: Date;
    isChanged: boolean;
}

File: API/Entities/Notification.cs
namespace API.Entities;

public class Notification
{
    public int Id { get; set; }
    public required string Title { get; set; }
    public required string Content { get; set; }
    public bool IsRead { get; set; }

    //Notifiction - AppUser
    public int SenderId { get; set; }
    public AppUser Sender { get; set; } = null!;
    public int RecipientId { get; set; }
    public AppUser Recipient { get; set; } = null!;
}
